src/main/java/com/growup/pms/task/domain/TaskUser.java

```diff
@@ -0,0 +1,44 @@
+package com.growup.pms.task.domain;
+
+import com.growup.pms.common.BaseEntity;
+import com.growup.pms.user.domain.User;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.ForeignKey;
+import jakarta.persistence.Id;
+import jakarta.persistence.IdClass;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
+
+@Entity
+@Getter
+@IdClass(TaskUserId.class)
+@Table(name = "task_users")
+@SQLDelete(sql = "UPDATE task_users SET is_deleted = true WHERE id = ?")
+@SQLRestriction(("is_deleted = false"))
```

Comment:
Typo: 괄호가 두 번 들어간 것 같습니다 :thinking: 

```suggestion
@SQLRestriction("is_deleted = false")
```

Reply:
수정했습니다 😅